### DISCUSSION TITLE
Population simulation for an unknown population.

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#965] [Questions & Problems] [answered]
Our population has not be defined in pksim, so i have simulated three populations in pksim and i want to choose the one that is closest to our population .
What is the selection criteria for that matter?
Your help is much appreciated .

**Comment by StephanSchaller** [comment]
Hi @ranamjz, can you elaborate a bit more on what you are trying to do?
DO you have data on the original population, biometrics? PK?

**Reply by ranamjz** [reply]
Thank you for answering.
Of course, i have an observed data set for the individuals attending the study.
But our population (ethnicity wise ) is not in the pksim data source . So i simulated 3 populations to see which one is the closest to our population ...but i do not know how to compare my individual observed data with the simulated population datas to see which one is more likely to be a match for my original population .

**Comment by msevestre** [comment, marked as answer]
See https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations

Section running and analyzing a population simulation.

**Reply by ranamjz** [reply]
Thank you so much for your help
